applications/OptimizationApplication/tests/process_tests/test_optimization_problem_vtu_output_process.py

```diff
@@ -0,0 +1,187 @@
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.model = Kratos.Model()
+        cls.model_part1 = cls.model.CreateModelPart("test_1")
+        cls.model_part1.CreateNewNode(1, 0.0, 0.0, 0.0)
+        cls.model_part1.CreateNewNode(2, 1.0, 0.0, 0.0)
+        cls.model_part1.CreateNewNode(3, 1.0, 1.0, 0.0)
+        properties = cls.model_part1.CreateNewProperties(1)
+        cls.model_part1.CreateNewElement("Element2D3N", 1, [1, 2, 3], properties)
+
+        for node in cls.model_part1.Nodes:
+            node.SetValue(Kratos.VELOCITY, Kratos.Array3([node.Id + 1, node.Id + 2, node.Id + 3]))
+
+        for element in cls.model_part1.Elements:
+            element.SetValue(Kratos.ACCELERATION, Kratos.Array3([element.Id + 1, element.Id + 2, element.Id + 3]))
+
+        cls.model_part2 = cls.model.CreateModelPart("test_2")
+        cls.model_part2.CreateNewNode(1, 0.0, 0.0, 0.0)
+        cls.model_part2.CreateNewNode(2, 1.0, 0.0, 0.0)
+        cls.model_part2.CreateNewNode(3, 1.0, 1.0, 0.0)
+        cls.model_part2.CreateNewNode(4, 0.0, 1.0, 0.0)
+        properties = cls.model_part2.CreateNewProperties(2)
+        cls.model_part2.CreateNewElement("Element2D4N", 1, [1, 2, 3, 4], properties)
+
+        for node in cls.model_part2.Nodes:
+            node.SetValue(Kratos.VELOCITY, Kratos.Array3([node.Id + 1, node.Id + 2, node.Id + 3]))
+
+        for element in cls.model_part2.Elements:
+            element.SetValue(Kratos.ACCELERATION, Kratos.Array3([element.Id + 1, element.Id + 2, element.Id + 3]))
+
+        cls.optimization_problem = OptimizationProblem()
+        cls.components_list = []
+
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyResponseFunction("resp_1", cls.model_part1))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyResponseFunction("resp_2", cls.model_part2))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyResponseFunction("resp_3", cls.model_part1))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyControl("control_1", cls.model_part1))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyControl("control_2", cls.model_part2))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyControl("control_3", cls.model_part1))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyExecutionPolicy("policy_1", cls.model_part1))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyExecutionPolicy("policy_2", cls.model_part2))
+        cls.components_list.append(TestOptimizationProblemVtuOutputProcess.DummyExecutionPolicy("policy_3", cls.model_part1))
+
+        for component in cls.components_list:
+            cls.optimization_problem.AddComponent(component)
+            ComponentDataView(component, cls.optimization_problem).SetDataBuffer(2)
+
+        # cls.components_list.append("algorithm")
```

Comment:
if the commented not needed please remove

Reply:
This will be removed :)

Reply:
done :)